specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/Resources.json

```diff
@@ -0,0 +1,105 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Resource Management events published to Azure Event Grid via Azure Resource Notifications",
+    "description": "Describes the schema of the Azure Resource Management events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "ResourceManagementCreatedOrUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ResourceNotifications.Resources.CreatedOrUpdated event.",
+      "type": "object",
+      "properties": {
+        "resourceInfo": {
+          "description": "Data specific to the resource",
+          "$ref": "#/definitions/ResourceInfoAboutCreatedOrUpdatedEvent"
+        },
+        "apiVersion": {
+          "description": "api version of the resource payload",
+          "type": "string"
+        }
+      },
+      "x-ms-examples": {
+        "ResourceManagementCreatedorUpdatedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/Resources_CreatedOrUpdated.json"
+        }
+      }
+    },
+    "ResourceInfoAboutCreatedOrUpdatedEvent": {
```

Comment:
Please either inline these properties (if we don't expect more events to use them), or use a base event as mentioned in a previous comment.